### DISCUSSION TITLE
Unusual use of /start or /fail or /{exit-status} or other?

### DISCUSSION THREAD

**Discussion by git70** [healthchecks/healthchecks#886] [Q&A] [answered]
Hi all!

I plan to run two simple scripts and run them as a system service (systemd on Debian 11). 
The first will send a ping when the computer starts up, the second when the computer is shut down.

Is it possible to somehow use `/start` or `/fail` or `/{exit-status}` (or another idea) to get the measured time of machine operation (between these pings)?

**Comment by cuu508** [comment, marked as answer]
Yes, you could send a `/start` when the computer starts up, and a success signal when it shuts down.

Healthchecks only calculates and displays the duration [if it is 72 hours or less](https://github.com/healthchecks/healthchecks/blob/75290f8f45bdf8761e67f1879dc026e34c97e56d/hc/api/models.py#L41) but perhaps that's sufficient for your use case?

Also now thinking about it, the 72 hour limit exists because the longer apart the start and the success signals are, the less confidence we have they are in fact related. _But_, if the client is using [Run IDs](https://blog.healthchecks.io/2022/11/using-run-ids-to-track-run-times-of-overlapping-jobs/), and the run IDs match, we know for sure they are related. And we could display the duration even if it is weeks or months.

**Reply by git70** [reply]
Thank you very much for the explanation!
I'm just starting my adventure with HC and the more I read about it, the more I'm delighted with this project :)
It is brilliantly designed and made!
